### ISSUE TITLE
Ideas for China and Japan

### ISSUE THREAD

**Issue by TinMann1** [Wolferos/Hearts-of-Iron-IV-The-Great-War#219]
Hello everyone,

I just wanted to say how much I am enjoying the mod. It is really an impressive project that you all should be proud of.

If you folks are on board with some of these ideas, I’d be happy to help expand the mod with some events for China and Japan and some other tweaks. I’ve played a number of run-throughs as both (they are my favs in the base game too), and I’ve noticed a few things that could be enhanced for the 1910 and 1914 scenarios:

1)	After the civil war China is very strong. For starters I think their military and industrial techs could be scaled back, or perhaps some modifiers from the base game could be adapted. 

2)	While China feels overpowered in terms of industry, Japan feels underpowered. China outpaces Japan almost immediately and by a significant margin, which I feel is unhistorical. This could be partially addressed by increasing the starting civilian factories in Japan as well as reducing those in China (as mentioned above.)

3)	China could have an event chain prior to the rebellion giving the Chinese player some choices. Perhaps the Qing could opt for more reforms (maybe have the option to evolve into a Constitutional Monarchy?) that might create pain but potentially put off the civil war. If the war happens and the Qing win, they could then opt for reforms or perhaps face a second revolution a few years later. Perhaps Japan could have an event to allow them to influence this second rebellion, or perhaps give them a CB to invade during the chaos.  

4)	If the Republic of China wins the civil war, there could be events that show how unstable and difficult to lead the nation was. For instance, the power vacuum left by Yuan Shikai’s abdication led to the warlord era starting around 1916. We could model that by creating a series of warlord states, perhaps allied together. Perhaps there could be an event chain that could put off the fracturing for a time, at a cost. Or perhaps it could be delayed if China is embroiled in a war. A series of small civil wars and uprisings would have the added benefit of keeping China’s army whittled down, and keep it from becoming immense so quickly.

5)	If it forms, the Republic of China may be best represented by an authoritarian state (rather than a republic). Granted there were elections and assemblies, but elections were rigged, ministers were stacked and dismissed arbitrarily, and by 1914 Yuan Shikai declared himself emperor before being forced to resign. In 1917 parliament was dissolved and General Zhang Zun appointed Puyi as emperor (again), which lead to another brief civil war. The country never had a chance for republican traditions to take hold. (Edit: after looking over the Xinhai Revolution event, the government type is critical for that event chain to work. So, this could be skipped. If others wanted to see this implemented, we could always do it after the war, maybe as part of one of the historical events mentioned above.)

6)	There could be an event chain that allows China to consider joining either side of WWI (or staying out), since it was hotly debated by the government.

7)	Japan could be given an option to attack China during this period, such as a hypothetical CB for the island of Hainan. In that example, if neither are in a faction, the war could be ended by event that allows China to surrender just the island—or they can fight it out to the bitter end. This would allow Japan to do something about China if it is clear they are becoming too powerful. Granted Japan should be wary since China my very well join the Entente, as they did historically. For this to work, Japan may want to attack China early, perhaps during the civil war, and before WWI breaks out. (Edit: This would have the added benefit of allowing Japan to choose a Fascist government through national focuses before WWI, and allow them to be more active in the game.)

8)	It would be nice if Japan could keep Korea as a puppet, say as a manpower pool. Another reason might be to have Korea remain neutral during a war with China, if the Japanese player didn’t want to fight along the Manchurian border. This might be handy if China joins the Entente and Japan the Central Powers. The focus tree now requires them to annex Korea to get to a wide swath of focuses. 

Thanks again for all of your hard work, let me know if any of these sound like something you’re interested in, and if you’d like help.

**Comment by TinMann1** [comment]
Hello again, I was feeling motivated today so I took a stab at a few items listed here. The attached sub-mod includes adjusted states to put China and Japan's industry on a more even footing. It also includes a short event chain where Japan could choose to intervene in the Xinhai Revolution in an optional, alternate history path. To start that event chain you will want to play as Japan from 1910. If you don't want to wait for the revolution event to fire, you can start the intervention manually though firing this event:

ww1_xinhai_intervention.200

Looking forward to feedback whenever time permits. Thanks in advance,

[TGW_Submod4 v0.2.zip](https://github.com/Wolferos/Hearts-of-Iron-IV-The-Great-War/files/4515559/TGW_Submod4.v0.2.zip)

**Comment by Wolferos** [comment]
Thanks a lot for your work and your time @TinMann1 ! We'll check and run tests on your modifications and if all goes well, we'll surely integrate them into the main branch in the future ;)

**Comment by TinMann1** [comment]
Of course! I'm happy to discuss these ideas in more depth anytime. Also, I'm making some slight tweaks and will re-post a v0.2 later today. thanks :)
(Edit: I have updated the submod files above. Also, if anyone tests the Japanese intervention, keep in mind Japan won't wind up with any land at the end because it's a civil war. The goal of intervening might be to get some army experience and perhaps choose focuses that require being at war. It's almost like being involved in the Spanish Civil War in the base game.)
